From 258863146f4ca86bae6ed4d21a3f5f4355207fb1 Mon Sep 17 00:00:00 2001
From: no92 <no92.mail@gmail.com>
Date: Sat, 9 Jul 2022 22:15:54 +0200
Subject: [PATCH] Add mlibc fixes

---
 client/pool-buffer.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/client/pool-buffer.c b/client/pool-buffer.c
index ea31edd..e1f5c1b 100644
--- a/client/pool-buffer.c
+++ b/client/pool-buffer.c
@@ -62,7 +62,7 @@ static struct pool_buffer *create_buffer(struct wl_shm *shm,
 	uint32_t stride = width * 4;
 	size_t size = stride * height;
 
-	char *name;
+	char *name = 0;
 	int fd = create_pool_file(size, &name);
 	assert(fd != -1);
 	void *data = mmap(NULL, size, PROT_READ | PROT_WRITE, MAP_SHARED, fd, 0);
-- 
2.36.1

