From 1bc096f343b4f35b4e5ab4c3efd3d62ad6165a33 Mon Sep 17 00:00:00 2001
From: czapek1337 <czapek1337@gmail.com>
Date: Mon, 9 May 2022 16:44:45 +0200
Subject: [PATCH] Accept linux-mlibc as a valid OS

---
 config.sub | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/config.sub b/config.sub
index 0f2234c..a09757d 100755
--- a/config.sub
+++ b/config.sub
@@ -131,7 +131,7 @@ case $1 in
 		# parts
 		maybe_os=$field2-$field3
 		case $maybe_os in
-			nto-qnx* | linux-gnu* | linux-android* | linux-dietlibc \
+			nto-qnx* | linux-gnu* | linux-android* | linux-dietlibc | linux-mlibc* \
 			| linux-newlib* | linux-musl* | linux-uclibc* | uclinux-uclibc* \
 			| uclinux-gnu* | kfreebsd*-gnu* | knetbsd*-gnu* | netbsd*-gnu* \
 			| netbsd*-eabi* | kopensolaris*-gnu* | cloudabi*-eabi* \
@@ -1355,7 +1355,7 @@ case $os in
 	     | chorusrdb* | cegcc* | glidix* \
 	     | cygwin* | msys* | pe* | moss* | proelf* | rtems* \
 	     | midipix* | mingw32* | mingw64* | linux-gnu* | linux-android* \
-	     | linux-newlib* | linux-musl* | linux-uclibc* \
+	     | linux-newlib* | linux-musl* | linux-uclibc* | linux-mlibc* \
 	     | uxpv* | beos* | mpeix* | udk* | moxiebox* \
 	     | interix* | uwin* | mks* | rhapsody* | darwin* \
 	     | openstep* | oskit* | conix* | pw32* | nonstopux* \
-- 
2.35.1

